proxy/README.md

```diff
@@ -0,0 +1,9 @@
+# proxy-example
+
+## What it does
+
+This extension provides boilerplate code to get started writing an addon which uses the Proxy API.
+
```

Comment:
I'd really like a bit more of a user-focused explanation of what this example does or why it might be useful. Even if it can't implement the whole use case, we could perhaps talk here about how it could be developed into something that addresses some use case or other.